projects/vdk-plugins/vdk-ingest-http/src/vdk/plugin/ingest_http/ingest_over_http.py

```diff
@@ -171,18 +174,20 @@ def __amend_payload(payload, destination_table):
     def __send_data(self, data, http_url, headers) -> IngestionResult:
         uncompressed_size_in_bytes = sys.getsizeof(data)
         compressed_size_in_bytes = None
+        data = json.dumps(data, allow_nan=self._allow_nan)
```

Comment:
Wouldn't `_allow_nan` be loaded as a string, and not as a boolean?

Reply:
By memory, would expect to be interpreted as boolean, since default value is boolean.
Also, did verify it locally, successfully (run a data job that enables allow nan).
And, we did elaborate on same thing with "INGEST_OVER_HTTP_VERIFY", that is boolean, and interpreted the same way currently

Reply:
Ok.